src/main/java/org/apache/sysml/api/DMLScript.java

```diff
@@ -188,15 +189,17 @@ public static void main(String[] args)
 		String[] otherArgs = new GenericOptionsParser(conf, args).getRemainingArgs();
 		
 		try {
-		DMLScript.executeScript(conf, otherArgs); 
+			DMLScript.executeScript(conf, otherArgs);
+		} catch (ParseException pe) {
+			System.err.println(pe.getMessage());
 		} catch (DMLScriptException e){
 			// In case of DMLScriptException, simply print the error message.
 			System.err.println(e.getMessage());
 		}
 	} 
 
 	public static boolean executeScript( String[] args ) 
-		throws DMLException
+		throws DMLException, ParseException
```

Comment:
I will create a JIRA to rename the exception and/or subclass DMLException since Matthias is currently making modifications to the exception hierarchy.